lockFileLint.js

```diff
@@ -1,52 +0,0 @@
-/*
```

Comment:
Do we need this for linting package-lock files?

Reply:
I don't think we would need it anymore, the `test:lockfile` script should be covering the single `yarn.lock`.
Previously we did need this script with all the different `package-lock.json` in each package.